projects/schematics/src/migrations/2_0/ssr/ssr.ts

```diff
@@ -298,28 +299,6 @@ export function updateAngularJsonFile(): Rule {
   };
 }
 
-async function checkIfSSRIsUsed(tree: Tree): Promise<boolean> {
```

Comment:
Why did you remove it for 2.0?

Reply:
Not removed, moved to a common location in the `projects/schematics/src/shared/utils/package-utils.ts`

Reply:
Good I thought it looked similar 😄